playground/LocaleUtils/index.js

```diff
@@ -20,6 +20,11 @@ export async function assertNoEnglishLeaks(mockFile, viewText, noTranslationCont
   */
   extractedString = extractedString.replace('《·', ' ');
   extractedString = extractedString.replace('《.', '《');
+  /*
+    Handle untranslated colon at the end of Odyssey screenreader text for field-level errors (reads as 'Error:')
+    TODO: Ask Odyssey why colon is not part of translation bundle
+  */
+  extractedString = extractedString.replace('《:', '《');
```

Comment:
Odyssey has opened a ticket to fix this, but here is a [SIW tracking ticket](https://oktainc.atlassian.net/browse/OKTA-689322)